exercises/sgf-parsing/canonical-data.json

```diff
@@ -0,0 +1,161 @@
+{
+    "exercise": "sgf-parsing",
+    "version": "1.0.0",
+    "cases": [
+        {
+            "description": "empty input",
+            "property": "parse",
+            "input": {
+                "encoded": ""
+            },
+            "expected": {
+                "error": "tree missing"
+            }
+        },
+        {
+            "description": "tree with no nodes",
+            "property": "parse",
+            "input": {
+                "encoded": "()"
+            },
+            "expected": {
+                "error": "tree no nodes"
+            }
+        },
+        {
+            "description": "node without tree",
+            "property": "parse",
+            "input": {
+                "encoded": ";"
+            },
+            "expected": {
+                "error": "tree missing"
```

Comment:
Maybe just use "node without tree"?

Reply:
I intentionally avoided mentioning the node because IMO it is much more straightforward to detect that the encoded input does not contain a valid tree than to determine that there is a node outside of a tree.